dolphinscheduler-dist/src/main/assembly/assembly-plugins.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+set -xeo pipefail
+
+DIST_DIR="$(pwd)/target"
+BIN_TAR_FILE="$DIST_DIR/apache-dolphinscheduler-*-bin.tar.gz"
+if [ ! -f $BIN_TAR_FILE ]; then
+  echo "$BIN_TAR_FILE not found!!!"
+  exit 1
+fi
+
+cd $DIST_DIR && tar -zxf apache-dolphinscheduler-*-bin.tar.gz
+cd $DIST_DIR/apache-dolphinscheduler-*-bin
+BIN_DIR=$(pwd)
+
+# move *-plugins/target/*-plugin/target/*.jar to *-plugins/
+PLUGINS_PATH=(
+alert-plugins
+datasource-plugins
+storage-plugins
+task-plugins
+)
+
+for plugin_path in ${PLUGINS_PATH[@]}
+do
+  cd $BIN_DIR/plugins/$plugin_path
+  find ./* -name "*.jar" | xargs -I {} mv {} ./
+  ls -d */ | xargs -I {} rm -rf {}
+done
```

Comment:
We should use maven assembly plugin to manage the plugin directory

Reply:
Assemby packaged directory:
- plugins
  - alert-plugins
    - dolphinscheduler-alert-dingtalk/taget/dolphinscheduler-alert-dingtalk-shade.jar

Directory processed with shell:
- plugins
  - alert-plugins
    - dolphinscheduler-alert-dingtalk-shade.jar

Assemby fuzzy matching directory can't remove the file path, so it needs secondary processing with shell.